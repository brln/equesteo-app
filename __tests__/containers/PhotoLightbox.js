import configureStore from 'redux-mock-store'
import React from "react"
import {fromJS, Map} from "immutable"
import renderer from "react-test-renderer"
import { Provider } from 'react-redux'
import thunk from "redux-thunk"

import PhotoLightboxContainer from '../../containers/PhotoLightbox'


const mockStore = configureStore([thunk])

test('render', () => {
  const store = mockStore(fromJS({
    pouchRecords: {},
    localState: {},
  }))

  const tree = renderer.create(
    <Provider store={store}>
      <PhotoLightboxContainer />
    </Provider>).toJSON()
  expect(tree).toMatchSnapshot()
})

