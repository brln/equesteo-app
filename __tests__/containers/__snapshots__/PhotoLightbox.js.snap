// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<View
  onLayout={[Function]}
  style={
    Object {
      "flex": 1,
      "overflow": "hidden",
    }
  }
>
  <View>
    <View
      style={
        Object {
          "zIndex": 9,
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "transparent",
            "opacity": 0,
          }
        }
      >
        <View
          style={
            Object {
              "bottom": 0,
              "justifyContent": "center",
              "left": 0,
              "position": "absolute",
              "top": 0,
              "zIndex": 13,
            }
          }
        >
          <View
            accessible={true}
            clickable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
          />
        </View>
        <View
          style={
            Object {
              "bottom": 0,
              "justifyContent": "center",
              "position": "absolute",
              "right": 0,
              "top": 0,
              "zIndex": 13,
            }
          }
        >
          <View
            accessible={true}
            clickable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
          />
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "transform": Array [
                Object {
                  "translateX": 0,
                },
              ],
              "width": 0,
            }
          }
        />
        <View
          style={
            Array [
              Object {
                "bottom": 0,
                "position": "absolute",
                "zIndex": 9,
              },
              Object {},
            ]
          }
        />
      </View>
    </View>
  </View>
</View>
`;
